Automattic/wp-calypso
# Pattern Library: Tweak mid-sized breakpoints

Fixes https://github.com/Automattic/dotcom-forge/issues/7716

## Proposed Changes

This PR makes it so the Pattern Library header doesn't change to a "mobile version" at such a wide breakpoint. Instead of 1280px, we now use 1080px.

## Why are these changes being made?


See p1717437000704179-slack-C048CUFRGFQ

## Testing Instructions



1. Try resizing the Pattern Library home page between 1440px and 960px and ensure that it looks good
2. Do the same for a category page
